### ISSUE TITLE
Clearly document which versions of IWXXM are required for which Annex 3 Amendment

### ISSUE THREAD

**Issue by braeckel** [wmo-registers/AvXML#26]
This could go in a Git repository documentation or on the WMO wiki, but is a more authoritative document needed?

**Comment by braeckel** [comment]
Agreed that this should be documented as a simple table on the WMO wiki as a sub-page off of https://wiswiki.wmo.int/tiki-index.php?page=TT-AvXML.

**Comment by blchoy** [comment]
I am not too sure if we need to have this completed before the release of IWXXM 2.1.1RC1, as we may need ICAO's view on the consequence in telling people that "the coding of XX in this and previous version of IWXXM should be the same".  While I don't mind having the table in place right now, we will definitely need to submit a paper on this for the upcoming WG-MIE meeting.  Therefore would it be more appropriate to move the milestone of this issue to the next WG-MIE meeting?

**Comment by braeckel** [comment]
I agree, I think this issue can be addressed after 2.1.1RC1 has been completed.  I am not sure whether WG-MIE input is needed, however, as there seems to be a clear relationship between IWXXM versions and Annex versions that can be documented by TT-AvXML.
